README.md

```diff
@@ -58,7 +58,25 @@ Role Variables
 
     - `interfaces`: a list of network interfaces to attach to the VM.
       Each network interface is defined with the following dict:
-        - `network`: Name of the network to which an interface should be attached.
+
+        - `type`: The type of the interface. Possible values:
+
+            - `network`: Attaches the interface to a named Libvirt virtual
+              network. This is the default value.
+            - `direct`: Directly attaches the interface to one of the host's
+              physical interfaces, using the `macvtap` driver.
+        - `network`: Name of the network to which an interface should be
+          attached. Must be specified if and only if the interface `type` is
+          `network`.
+        - `source_interface`: A dict defining the host interface to which this
```

Comment:
nit: I wonder if we should try to stick as close to the XML format as possible. There's no 1:1 mapping between YAML and XML, but if we s/source_interface/source/ and s/name/dev/ it could be close.

Reply:
Sure.